F/FFLAS_FFPACK/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FFLAS_FFPACK"
+version = v"2.5.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/linbox-team/fflas-ffpack/releases/download/v$version/fflas-ffpack-$version.tar.gz", "dafb4c0835824d28e4f823748579be6e4c8889c9570c6ce9cce1e186c3ebbb23"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/fflas-ffpack-*
+
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+
+autoreconf
+./configure CCNAM=${CC} CPLUS_INCLUDE_PATH=$includedir --prefix=$prefix --build=${MACHTYPE} --host=${target}
+
+make -j ${nproc}
+make install
+
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(exclude=Sys.iswindows) |> expand_cxxstring_abis
+
+# Strangely, libblastrampoline does not seem to be there on aarch64-unknown-freebsd
+filter!(p->!(os(p)=="freebsd" && arch(p)=="aarch64"), platforms)
+
+# The products that we will ensure are always built
+products = [
+    FileProduct("include/fflas-ffpack/fflas-ffpack-config.h", :fflas_ffpack_config_h)
+]
```

Comment:
Is the product only a header file?

Reply:
No, this is almost-pure C++ library, which produces lots of files, all in include/ except for a bin/fflas-ffpack-config. Should I include more?

Reply:
After opening the archive and not seeing any binary product I believe the correct answer should have been: yes, this is a header-only package.